### PR TITLE
Create RELEASE.md

### PR DESCRIPTION
**Pull request by roryhardy**
Added the release guide

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How To Release
+
+This project is hosted on NPM.  You can see it [here][project-url].
+
+Releasing the project requires these steps:
+
+0. Set the version number in the code
+1. `npm run build`
```
Comment (mhemesath) [inline RELEASE.md]: I should update the prepublish to run build then we can omit this step

Comment (roryhardy) [inline RELEASE.md]: Do you want me to omit it now?

Comment (mhemesath) [inline RELEASE.md]: Yeah, just omit it and we'll update the prepublish

**Comment by bjankord** [comment]
+1

**Comment by mhemesath** [comment]
👍

**Pull request merged**